src/modules/pair/services/pair.compute.service.ts

```diff
@@ -28,6 +35,11 @@ export class PairComputeService implements IPairComputeService {
         private readonly dataApi: MXDataApiService,
         private readonly analyticsQuery: AnalyticsQueryService,
         private readonly apiConfig: ApiConfigService,
+        private readonly farmAbi: FarmAbiFactory,
```

Comment:
don't use the farmAbiFactory in here. Because we are working only with active V2 farms, it's safe to use directly the farmAbiV2

Reply:
done